### DISCUSSION TITLE
Asset management

### DISCUSSION THREAD

**Discussion by ajthinking** [orchestral/testbench-dusk#57] [Q&A]
Thanks for this wonderful tool!

I'm making a package with js/css assets. Normally the user would publish these so they end up in dir 'public/vendor/something'.
But Im not sure how to set that up with the testbench. Is there a way to fake it so they will be accessible at url '/vendor/x/something.css' for instance?

**Comment by ajthinking** [comment]
Found a way by using `$path = $this->getBasePath() ` and copying my files into `$path/public/vendor/package/x`

**Comment by crynobone** [comment]
You can use:

```
./vendor/bin/testbench-dusk vendor:publish
```
